### DISCUSSION TITLE
Make inserted images resizable

### DISCUSSION THREAD

**Discussion by Wehaveall** [slab/quill#4455] [Ideas]
There was a plugin to. 1.3.6, but it don't work anymore. This ans emojis are the only features I miss.

**Comment by VaclavKosak** [comment]
@luin Do we have any update for this?

**Reply by Wehaveall** [reply]
Please...we neeed this feature so much...

**Comment by enzedonline** [comment]
@Wehaveall @VaclavKosak  [Blotformatter2](https://github.com/enzedonline/quill-blot-formatter2) is Quill 2 friendly

<img width="448" height="198" alt="resize" src="https://github.com/user-attachments/assets/dfd7b547-7fb2-4477-be88-e4571892d636" />
